include/ibamr/INSVCStaggeredConservativeHierarchyIntegrator.h

```diff
@@ -169,6 +169,11 @@ class INSVCStaggeredConservativeHierarchyIntegrator : public INSVCStaggeredHiera
      */
     void registerMassDensitySourceTerm(SAMRAI::tbox::Pointer<IBTK::CartGridFunction> S_fcn);
 
+    /*!
+     * Returns the number of cycles to perform for the present time step.
+     */
+    int getNumberOfCycles() const;
```

Comment:
This is already implemented in the parent class. Why does it need to be reimplemented here?

Reply:
The conservative convective class does not use standard IBAMR time stepping MIDPOINT, EULER etc. The parent class checks that the convective scheme adheres to one of the above and faults when not. Hence, we override.